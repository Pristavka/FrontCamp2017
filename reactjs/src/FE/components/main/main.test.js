import React from 'react';
import Main from './main';

it('Should render a Main', () => {
  const wrapper = shallow(<Main />);
  expect(wrapper).toMatchSnapshot();
});

