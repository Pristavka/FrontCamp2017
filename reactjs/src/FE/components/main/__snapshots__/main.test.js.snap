// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Main 1`] = `
<React.Fragment>
  <Header />
  <div
    className="wrapper"
  >
    <div
      className="content"
    >
      <Connect(PostsList) />
    </div>
  </div>
</React.Fragment>
`;
